lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -30,11 +30,11 @@ module Billing #:nodoc:
     class OrbitalGateway < Gateway
       include Empty
 
-      API_VERSION = "5.6"
+      API_VERSION = "7.1"
 
       POST_HEADERS = {
         "MIME-Version" => "1.1",
-        "Content-Type" => "application/PTI56",
+        "Content-Type" => "application/PTI#{API_VERSION.gsub(/\./, '')}",
```

Comment:
Just out of curiosity- do they list their API version as `7.1` but are expecting it sent as `PTI71`? Just wondering if we should set the `API_VERSION = "PTI71"` instead.